catch/catch.js

```diff
@@ -26,7 +26,7 @@ img1Block.addEventListener('click', () => {
     incrementEncounter(currentThreeArray[2]);
 
     if (Number(getFromLocalStorage('caught') === 10)) {
-        window.location.href = './results/results.html';
+        window.location.replace('../results/results.html');
```

Comment:
seems to not be routing properly on your deployed site still.  I'm unsure if this is the problem, but what if we changed the file name of your results html from 'results.html' to 'index.html'. Then the pathing will be '../results/index.html'